plugins/localwatchdog.d/20_sysinfo

```diff
@@ -19,7 +20,8 @@ function WatchFile ()
     FILE=$1
 
     TMPFILE=$(mktemp -p /mnt/testarea -t Watch.XXXXX)
-    while true; do
+    SECONDS=0
+    while [ $SECONDS -lt $MAX_SECS_IN_LOOP ]; do
```

Comment:
I must be blind because I can't see where SECONDS is incremented.  ??

Reply:
@StykMartin pointed out the special variable SECONDS.  I thought that would be it, but wasn't sure and better to ask and be sure then to miss something. :-)

Reply:
SECONDS is a magic bash variable.  I'm trying to find a decent google description.
I added you to this changeset because I'm worried a customer had requested a special Start/Stop labeling around each sysrq-trigger request and we could be breaking something by getting rid of it.  Instead I surround all 3 sysrq-trigger requests and wait for that to complete but only for a given period of time.  As Martin mentioned, best effort on producing this output. The real drive to this change is: if there is a failure in this plugin, it's very difficult to determine whether the fault is restraint or the user's task.

Reply:
This is a good description.  I can create my own variable and increment if this usage concerns anyone.
https://www.oreilly.com/library/view/shell-scripting-expert/9781118166321/c03-anchor-3.xhtml

Reply:
I would suggest to make a comment about SECONDS in the code. Seems okay to use. But document how it is special and automatically increments.

Reply:
I've added your requested comments.